<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BinUtilsDialog</class>
 <widget class="QDialog" name="BinUtilsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="objdumpLineEdit">
     <property name="placeholderText">
      <string>objdump</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="elfButton">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="continueButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="elfLineEdit">
     <property name="placeholderText">
      <string>program.elf</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="objdumpButton">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>objdumpLineEdit</tabstop>
  <tabstop>objdumpButton</tabstop>
  <tabstop>elfLineEdit</tabstop>
  <tabstop>elfButton</tabstop>
  <tabstop>continueButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
